mythril/laser/ethereum/call.py

```diff
@@ -179,16 +180,26 @@ def get_call_data(
     )
 
     try:
-        calldata_from_mem = state.memory[
-            util.get_concrete_int(memory_start) : util.get_concrete_int(
-                memory_start + memory_size
+        uses_entire_calldata = False
+        try:
+            uses_entire_calldata = simplify(
+                memory_size - global_state.environment.calldata.calldatasize == 0
             )
-        ]
-        call_data = ConcreteCalldata(transaction_id, calldata_from_mem)
-        log.debug("Calldata: " + str(call_data))
+        except Z3Exception:
```

Comment:
I can't get the reason why Z3Exception is caught here? can you explain.

Reply:
:sweat_smile: A previous version had `if simplify(...): `, which would raise an exception. It's unneeded now.